server/node.go

```diff
@@ -238,6 +241,10 @@ func (n *Node) start(rpcServer *rpc.Server, engines []engine.Engine,
 		log.Fatalf("unable to register node service with RPC server: %s", err)
 	}
 
+	// Start status monitor.
+	sub := n.ctx.EventFeed.Subscribe()
+	go n.status.StartMonitorFeed(sub)
```

Comment:
Perhaps I missedread how this flow works.  Is there a reason to call this using a separate thread?